docs/preview/features/eventhubs-worker-template.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Azure EventHubs worker template"
+layout: default
+---
+
+# Azure EventHubs Worker Project Template
+
+## Create Your First Arcus Azure EventHubs Worker Project
+
+First, install the template from NuGet:
+
+```shell
+> dotnet new --install Arcus.Templates.EventHubs
+```
+
+When installed, the template can be created with shortname: `arcus-eventhubs`:
+
+```shell
+> dotnet new arcus-eventhubs --name Arcus.Demo.EventHubs 
+```
+
+## Features
+
+Creates a starter worker project with by default configured:
+
+* TCP health check probe ([official docs](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-2.2) & [Arcus docs](https://messaging.arcus-azure.net/features/tcp-health-probe)).
+* Example message pump on Azure EventHubs ([Arcus EventHubs messaging docs](), [Microsoft EventHubs official docs](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues))
```

Comment:
Frontend build fails bc messaging docs is not yet released, so can't add it here yet.
Waiting for https://github.com/arcus-azure/arcus.messaging/pull/337

Reply:
Updates this.